content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -117,7 +117,7 @@ In addition to RippleAPI-specific code, this script uses syntax and conventions
 const RippleAPI = require('ripple-lib').RippleAPI;
 ```
 
-The opening line enables [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). This is purely optional, but it helps you avoid some common pitfalls of JavaScript. See also: [Restrictions on Code in Strict Mode](https://msdn.microsoft.com/library/br230269%28v=vs.94%29.aspx#Anchor_1).
+The opening line enables [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode). This is purely optional, but it helps you avoid some common pitfalls of JavaScript.
```

Comment:
Microsoft appears to have redirected their Strict Mode page to the Mozilla page already linked in this line, so I removed the Microsoft link.